R/getCoords.R

```diff
@@ -1,36 +1,37 @@
 #' Get Coordinates
-#' 
+#'
 #' Extracts spatial coordinates of a MAgPIE-object
-#' 
-#' 
-#' @aliases getCoords getCoords<- 
+#'
+#'
+#' @aliases getCoords getCoords<-
 #' @param x MAgPIE object
 #' @param xlab label of x-dimension
 #' @param ylab label of y-dimension
-#' @param value coordinates as two column data.frame the data should be set to 
+#' @param value coordinates as two column data.frame the data should be set to
 #' (first column = x, second column = y).
 #' @return coordinates of the MAgPIE-object
 #' @author Jan Philipp Dietrich
 #' @seealso \code{\link{as.RasterBrick}}, \code{\link{getItems}}, \code{"\linkS4class{magpie}"}
 #' @examples
-#'   a <- maxample("animal")#'   
-#'   getCoords(a)
-#' 
+#' a <- maxample("animal")
+#' getCoords(a)
 #' @export
-getCoords <- function(x, xlab="x", ylab="y") {
-  if(!hasCoords(x,xlab,ylab)) stop("No coordinates found in object!")
-  .tmp <- function(x,n) return(as.numeric(sub("p",".", getItems(x,n,full=TRUE), fixed=TRUE)))
-  return(data.frame(x=.tmp(x,xlab),y=.tmp(x,ylab)))
+getCoords <- function(x, xlab = "x", ylab = "y") {
+  if (!hasCoords(x, xlab, ylab)) stop("No coordinates found in object!")
+  .tmp <- function(x, n) return(as.numeric(sub("p", ".", getItems(x, n, full = TRUE), fixed = TRUE)))
+  return(data.frame(x = .tmp(x, xlab), y = .tmp(x, ylab)))
```

Comment:
This almost looks like code golf, I'd definitely appreciate some extra newlines and `{}` here ;)